pkg/recipes/controllerconfig/config.go

```diff
@@ -76,6 +76,14 @@ func New(options hostoptions.HostOptions) (*RecipeControllerConfig, error) {
 		return nil, err
 	}
 
+	if options.Config.Bicep.DeleteRetryCount == "" {
+		options.Config.Bicep.DeleteRetryCount = "3"
```

Comment:
Should these be constants at the top of the file?

Reply:
If we end up needing these in multiple places we can make that change.